src/test/java/school/lemon/changerequest/java/generics/numberGenerators/DoubleGeneratorTestUnit.java

```diff
@@ -0,0 +1,22 @@
+package school.lemon.changerequest.java.generics.numberGenerators;
+
+import org.junit.*;
+
+/**
+ * Created by User on 03.01.2017.
+ */
+public class DoubleGeneratorTestUnit {
```

Comment:
There is no need to point that it's Unit test in class name, it should be just: `DoubleGeneratorTest`